pkg/model/serializer.go

```diff
@@ -437,6 +444,37 @@ func (v ExternalVolume) MarshalYAML() (interface{}, error) {
 	return v.Name + ":" + v.SubPath + ":" + v.MountPath, nil
 }
 
+// UnmarshalYAML Implements the Unmarshaler interface of the yaml pkg.
+func (healthcheckProbes *HealthchecksProbes) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var rawBool bool
+	err := unmarshal(&rawBool)
+	if err == nil {
+		healthcheckProbes.Liveness = rawBool
+		healthcheckProbes.Startup = rawBool
+		healthcheckProbes.Readiness = rawBool
+		return nil
+	}
+
+	var healthCheckProbesRaw healthCheckProbesRaw
+	err = unmarshal(&healthCheckProbesRaw)
+	if err != nil {
+		return err
+	}
+
+	healthcheckProbes.Liveness = healthCheckProbesRaw.Liveness
+	healthcheckProbes.Startup = healthCheckProbesRaw.Startup
+	healthcheckProbes.Readiness = healthCheckProbesRaw.Readiness
+	return nil
+}
+
+// MarshalYAML Implements the marshaler interface of the yaml pkg.
+func (healthcheckProbes HealthchecksProbes) MarshalYAML() (interface{}, error) {
```

Comment:
We should print `probes` if any of them is true. false values won't be printed

Reply:
If any of the probes is true it will be printed in the health checks field, if all the probes are enabled the healthchecks field will be set to true. If none of the probes are true then it won't print anything